tutorial/index.html

```diff
@@ -0,0 +1,412 @@
+        simplified
+        the functionality a bit to keep the focus on the essentials.
+    </p>
+
+    <h3>Posting tweets</h3>
+
+    <p>
+        You can post tweets via a simple REST endpoint. A tweet consists of a username and the message. We use
+        <strong>Akka Persistence</strong> to keep track of the tweets for each user and be able to recover the
+        timeline for a user after a restart of the application. All incoming tweets are put on the event bus that comes
+        with <strong>Akka</strong> once they are persisted.
+    </p>
+
+    <h3>Streaming tweets via Websockets</h3>
+
+    <p>
+        On the other side an actor is subscribed to the event bus. This means the actor will be notified every time a
+        new tweet is put on the event bus. Since <strong>Akka</strong> implements the <a
+            href="http://www.reactive-streams.org/announce-1.0.0">
+        Reactive Streams</a> SPI we can turn the actor into a <strong>Publisher</strong> which we can then turn into a
+        <strong>Source</strong> so we can use it in an <strong>Akka Http Flow</strong>.
+    </p>
+
+    <h3>The same story but now in a nice diagram</h3>
+    <img src="tutorial/img/flow.png" alt="The complete flow from posting a tweet to it being pushed on the websocket">
+    <caption>The complete flow from posting a tweet to it being pushed on the websocket</caption>
+    <p>
+        Next: <a href="#tutorial/2" class="shortcut">Posting a Tweet</a>
+    </p>
+</div>
+
+<div>
+    <h2>Part 1: Posting a Tweet</h2>
+
+    <p>
+        We start in <a href="#code/src/main/scala/reactive/Main.scala" class="shortcut">Main.scala</a> which defines the
+        routes. The route (in this case a REST endpoint) for posting a tweet is defined in the following method:
+
+<pre><code>def addTweet = {
+  post {
+    entity(as[Tweet]) { tweet =>
+      complete {
+        (system.actorOf(TweetActorManager.props) ? tweet).map(_ => StatusCodes.NoContent)
+      }
+    }
+  }
+}</code></pre>
+
+    The interesting part resides within the <code>complete</code> directive. There we create an actor (<a
+        href="#code/src/main/scala/reactive/tweets/incoming/TweetActorManager.scala"
+        class="shortcut">TweetActorManager</a>)
+    and send it the tweet. For simplicity we ignore failure and transform a successful completion to an appropriate
+    HTTP status code.
+    </p>
+    <p>
+        Next: <a href="#tutorial/3" class="shortcut">Using Akka Persistence</a>
+    </p>
+</div>
+
+<div>
+    <h2>Part 1: Using Akka Persistence</h2>
+
+    <h3>Forwarding tweets to persistent actors</h3>
+
+    <p>
+        All posted tweets end up at the <a href="#code/src/main/scala/reactive/tweets/incoming/TweetActorManager.scala"
+                                           class="shortcut">TweetActorManager</a>.
+        This actor forwards the tweet to a persistent actor based on the user that posted the tweet. Because we
```

Comment:
"a persistent actor" -> the actor itself is persistent? Or do you mean a "persistence actor"?